xarray/tests/test_backends.py

```diff
@@ -1389,6 +1408,10 @@ def roundtrip(self, data, save_kwargs={}, open_kwargs={},
                   allow_cleanup_failure=False):
         yield data.chunk()
 
+    def test_roundtrip_string_encoded_characters(self):
+        # Override method in DatasetIOTestCases - not applicable to dask
+        pass
```

Comment:
should we use `pytest.skip()` here?

Reply:
I'm not sure this is a skipped test, so much as a test that really shouldn't exist at all :). We slightly abuse the notion of a "backend" for dask here.